cumulusci/robotframework/CumulusCI.py

```diff
@@ -269,14 +284,20 @@ def end_perf_timer(self):
     def set_test_elapsed_time(self, elapsedtime):
         """This keyword captures a computed rather than measured elapsed time for performance tests.
 
-        For example, if you were performance testing a Salesforce batch process, you might want to store the Salesforce-measured elapsed time of the batch process instead of the time measured in the CCI client process.
+        For example, if you were performance testing a Salesforce batch process, you might want to
+        store the Salesforce-measured elapsed time of the batch process instead of the time measured
+        in the CCI client process.
 
         The keyword takes a single argument which is either a number of seconds or a Robot time string
         (https://robotframework.org/robotframework/latest/libraries/DateTime.html#Time%20formats).
 
         Using this keyword will automatically add the tag cci_metric_elapsed_time to the test case
```

Comment:
You should mention that this tag will not appear in the list of tag statistics, otherwise people might be confused as to why it's not there. At least, people like me were confused by that :-)

Reply:
I added: "cci_metric_elapsed_time is not included in Robot's html statistical roll-ups."